### PR TITLE
Fixed 'Confirm Mnemonic' form in case of repeating word in generated mnemonic

### PR DESCRIPTION
**Pull request by ozalexo**

### CODE REVIEW

File: packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js [hunk 1]
```diff
@@ -95,17 +92,23 @@ export default class ConfirmMnemonicContainer extends Component {
     )
   }
 
+  checkIfWordAlreadyUsed = (wordItem) =>
+    this.state.confirmPhrase
+      .findIndex((item) => item.index === wordItem.index) !== -1 // Not found by index equals to "word is not used yet"
```
Comment (e-maximus) [inline packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js]: I'd add '{' braces around this function to get a better understanding. Now it's look like part of another method

File: packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js [hunk 2]
```diff
@@ -49,14 +49,11 @@ export default class ConfirmMnemonicContainer extends Component {
...
-      this.setState(
-        { confirmPhrase: this.state.confirmPhrase.concat(word) },
-        this.refreshCurrentMnemonicForm
-      )
-      return true
-    }
+  handleClickWord = (wordItem) => {
+    this.setState(
+      { confirmPhrase: this.state.confirmPhrase.concat(wordItem) },
+      this.refreshCurrentMnemonicForm
```
Comment (e-maximus) [inline packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js]: Why we name it "refreshCURRENTmnemonicForm"? Is there any other Mnemonic form at the same time? It's better to remove current word from param name

File: packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js (hunk 1 above)
Comment (ozalexo) [reply packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js]: Done

File: packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js (hunk 2 above)
Comment (ozalexo) [reply packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js]: Done

**Pull request merged**